themes/default/content/docs/intro/cloud-providers/azure/_index.md

```diff
@@ -121,6 +121,15 @@ The following packages are available in package managers:
 
 The native Azure provider SDKs are open source and available in the [pulumi/pulumi-azure-native](https://github.com/pulumi/pulumi-azure-native) repo.
 
+## Easy SDK Integration
+
+The native Azure provider SDKs provide convenience helpers to hydrate an Azure SDK client in each supported language seeded with the provider's configured credentials. This allows users to operate on resources that are either not serviced by the Azure Resource Manager (ARM) resource model or are not compatible with the lifecycle for a Pulumi resource. See below for links to examples in each of the supported languages:
```

Comment:
> not serviced by the Azure Resource Manager (ARM) resource model

I suspect the token that we return may not work for other APIs (e.g., data plane operations).